cryoet_data_portal_neuroglancer/state_generator.py

```diff
@@ -198,8 +198,7 @@ def generate_image_layer(
     Note, if twodee_contrast_limits are not provided, the contrast limits will be calculated using the mean and rms values. If threedee_contrast_limits are not provided, the contrast limits will be the same as the twodee_contrast_limits.
     """
     # If volume rendering is visible, set the flag to True for the relevant shader
-    if not has_volume_rendering_shader and volume_rendering_is_visible:
-        has_volume_rendering_shader = True
+    has_volume_rendering_shader = volume_rendering_is_visible or threedee_contrast_limits is not None
```

Comment:
The `has_volume_rendering_shader` should keep it's value if it is not None (i.e. it was manually set to False or True).

Is there a different idea you would suggest for this? To overall idea is to basically have it be auto, in which case it can use the other parameters to come up with the setting that "makes sense"

Or you can manually enable/disable it.